project-fortis-interfaces/src/components/Admin/UserRoles.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import Snackbar from 'material-ui/Snackbar';
+import groupBy from 'lodash/groupBy';
+import differenceBy from 'lodash/differenceBy';
+import { DataGrid } from './DataGrid';
+import { getColumns } from './shared';
+const { Editors, Formatters } = require('react-data-grid-addons');
+const { DropDownEditor } = Editors;
+const { DropDownFormatter } = Formatters;
+
+const roles = ['user', 'admin'];
+
+class UserRoles extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false,
+      message: ''
+    };
+
+    this.getUserRolesColumns = this.getUserRolesColumns.bind(this);
+    this.handleSave = this.handleSave.bind(this);
+    this.handleRemove = this.handleRemove.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.flux.actions.ADMIN.load_users()
+  }
+
+  getUserRolesColumns() {
+    const columnValues = [
+      {editable: true, filterable: true, sortable: true, key: "identifier", name: "Identity"},
```

Comment:
Should this be `editable: false` since we only support adding/deleting users?

Reply:
If I do this, then you won't be able to initialize a user's name (when you click on add user it'll generate an empty row, so you won't be able to input the identifier) :(